### DISCUSSION TITLE
Average Storage Space Per Screenshot

### DISCUSSION THREAD

**Discussion by erikmakela** [openrecall/openrecall#78] [General]
Hi,

I'm curious, what is the average storage space per screenshot?

**Comment by c00ldude1oo** [comment]
It's about 700 KB. I have 51k screenshots at 1080p using 31 GB of storage.

**Comment by erikmakela** [comment]
@c00ldude1oo Thank you. Would you happen to know if there's a way to change the interval at which screenshots are taken?

**Reply by c00ldude1oo** [reply]
It doesn't look like it. It does compare each screenshot to the last and if its very similar it skips saving the image.

**Reply by erikmakela** [reply]
Thank you again. I do wonder if the similarity metric can be change. I'll look into it later if needed.
